### PR TITLE
Update systems.csv - PBSC cities

### PR DESCRIPTION
**Pull request by maduprasPBSC**
Added new PBSC cities:
* Porto Alegre
* Rio de Janeiro
* Salvador
* São Paulo
* Vila Velha
* Nicosia
* Tucson

Modified:
* Recife: clarified the system name

### CODE REVIEW

File: systems.csv [hunk 1]
```diff
@@ -11,7 +11,12 @@ AW,Greenbike Aruba,Aruba,greenbike_aruba,http://greenbikearuba.com,https://aru.p
...
 BA,Sarajevo Bosnia and Herzegovina,"Sarajevo, BA",nextbike_ba,https://www.nextbike.ba/xx/sarajevo/,https://api.nextbike.net/maps/gbfs/v1/nextbike_ba/gbfs.json
 BA,Tuzla,"Tuzla, BA",nextbike_ct,https://www.nextbike.ba/xx/tuzla/,https://api.nextbike.net/maps/gbfs/v1/nextbike_ct/gbfs.json
-BR,Bike Itaú,"Recife, BR",bike_itau,https://bikeitau.com.br/,https://rec.publicbikesystem.net/ube/gbfs/v1/
+BR,Bike Itaú - Pernambuco,"Recife, BR",bike_itau,https://bikeitau.com.br/bikepe,https://rec.publicbikesystem.net/ube/gbfs/v1/
+BR,Bike Itaú - Poa,"Porto Alegre, BR",bike_itau_poa,https://bikeitau.com.br/bikepoa,https://poa.publicbikesystem.net/ube/gbfs/v1/
+BR,Bike Itaú - Rio,"Rio de Janeiro, BR",bike_itau_rio,https://bikeitau.com.br/bikerio/,https://riodejaneiro.publicbikesystem.net/ube/gbfs/v1/
+BR,Bike Itaú - Salvador,"Salvador, BR",bike_itau_salvador,https://bikeitau.com.br/bikesalvador/,https://salvador.publicbikesystem.net/ube/gbfs/v1/
+BR,Bike Itaú - Sampa,"São Paulo, BR",bike_itau_sampa,https://bikeitau.com.br/bikesampa,https://saopaulo.publicbikesystem.net/ube/gbfs/v1/
+BR,Bike VV,"Vila Velha, BR",bike_vv,https://www.bikevv.com.br/,https://vilavelha.publicbikesystem.net/ube/gbfs/v1/
```
Comment (jcn) [inline systems.csv]: These system_ids all seem to be wrong (they don't include the `itau` identifier in the actual `system_information` feed when I fetch it).

File: systems.csv [hunk 2]
```diff
@@ -20,6 +25,7 @@ CA,Sobi Hamilton,Hamilton Ontario,sobi_hamilton,https://hamilton.socialbicycles.
 CA,VeloGo,"Ottawa, ON",velgo,http://velogo.ca/,http://velogo.ca/opendata/gbfs.json
 CH,nextbike Switzerland,"CH",nextbike_ch,https://www.nextbike.ch/xx/sursee/,https://api.nextbike.net/maps/gbfs/v1/nextbike_ch/gbfs.json
 CY,nextbike Cyprus,"CY",nextbike_cy,https://www.nextbike.com.cy/xx/limassol/,https://api.nextbike.net/maps/gbfs/v1/nextbike_cy/gbfs.json
+CY,Velespeed,"Nicosia, CY",velespeed,https://en.velespeed.net/,https://nicosia.publicbikesystem.net/ube/gbfs/v1/
```
Comment (jcn) [inline systems.csv:28]: Why doesn't this system have a `system_id` listed in its `system_information` (it's blank)?

File: systems.csv [hunk 3]
```diff
@@ -202,5 +208,6 @@ US,Spartanburg BCycle,"Spartanburg, SC",bcycle_spartanburg,https://spartanburg.b
 US,Spin DC,"Washington, DC",spin,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/gbfs
 US,START Bike,"Jackson, WY",bcycle_startbike,https://startbike.bcycle.com,https://gbfs.bcycle.com/bcycle_startbike/gbfs.json
 US,Topeka Metro Bikes,"Topeka, KS",topeka_metro_bikes,http://topekametrobikes.org/,http://topekametrobikes.org/opendata/gbfs.json
+US,Tugo,"Tucson, AZ",tugo,https://tugobikeshare.com/,https://tuc.publicbikesystem.net/ube/gbfs/v1/
```
Comment (jcn) [inline systems.csv:211]: This system_id is also blank...

**Review by jcn — COMMENTED** [review]
The feeds all resolve, so this is close to correct, but the actual system_ids are all wrong (the BR ones are wrong, and the other two just aren't in the actual feed file). Is this something that can be corrected on your side? I'm just worried this will cause issues with people who count on these values to match up between the `systems.csv` file and what's actually in the feeds.

File: systems.csv (hunk 2 above)
Comment (maduprasPBSC) [reply systems.csv:28]: The system id for Nicosia, Vila Velha and Tucson are planned to be added in the 4.8.5 version of the PBSC Backend

**Comment by maduprasPBSC** [comment]
Nicosia, Vila Velha and Tucson system id are now properly returned

**Comment by jcn** [comment]
> Nicosia, Vila Velha and Tucson system id are now properly returned

Thanks!

**Pull request merged**